go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/prometheus/client_golang v1.4.0
 	github.com/rboyer/safeio v0.2.1
 	github.com/ryanuber/columnize v2.1.0+incompatible
-	github.com/shirou/gopsutil v2.20.6-0.20200630091542-01afd763e6c0+incompatible
+	github.com/shirou/gopsutil v2.20.9+incompatible
```

Comment:
Is the `+incompatible` flag a concern here? What does it mean?

Reply:
It means that though the module advertises its version is v2, it doesn't file the rest of the Go Modules rules for v2.